src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -138,17 +149,28 @@ class AddressValidationModal extends React.Component {
     const showEditLinkErrorState = addressValidationError && validationKey;
     const showEditLinkNonErrorState = !addressValidationError;
     const showEditLink = showEditLinkErrorState || showEditLinkNonErrorState;
+    const showFirstRadioButton =
```

Comment:
I feel like naming this something like `showRadioButton` would be a little clearer.

Reply:
But this is only for the first radio button, not all radio buttons

Reply:
Renamed it and used it as a condition for the onclick handler as well